### ISSUE TITLE
Update .ccls upon installing a new library

### ISSUE THREAD

**Issue by anurag3301** [platformio/platformio-core#4950]
What kind of issue is this?

- [x] **Feature Request**.
------------------------------------------------------------------

### Configuration

**Operating system**: Linux arch 6.9.7-arch1-1 \#1 SMP PREEMPT_DYNAMIC Fri, 28 Jun 2024 04:32:50 +0000 x86_64 GNU/Linux

**PlatformIO Version** (`platformio --version`): PlatformIO Core, version 6.1.15

### Description of problem
I use neovim along with ccls, upon initializing the project using the `--ide vim` flag it creates a `.ccs` file which works perfectly. However upon installing a new library using `pio pkg install` it does not update the `.ccls` file with the include path of the library's header file, due to that the language server has no idea about the library headers which makes the development unpleasant.

I think it should be easy to implement, the library is already included while building, so I think I should be able to add the `-I/path/` in the .ccls file for the library. 

I'd like to work on this, can you guild be a bit about which files I should look into?

#### Steps to Reproduce

1. Create a project with `--ide vim` flag
2. Check the `.ccls` file 
3. Add a new library using `pio pkg install`
4. Check the language server is not able to find the library's includes as it's not there in the `.ccls` file

### Actual Results
Update the .ccls file with the include path for library

### Expected Results
Doesn't the .ccls file with the include path for the library

**Comment by Tyrn** [comment]
Is there a way to update `.ccls` file manually?

**Comment by anurag3301** [comment]
Yes you can manually update the `.ccls` file, its basically a makefile so you add your own `-D` and `-I` flags
